server/aws.js

```diff
@@ -0,0 +1,48 @@
+const AWS = require('aws-sdk');
+
+AWS.config.update({
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+});
+
+const sqs = new AWS.SQS({ region: 'us-west-1' });
+
+const inventoryMessage = { topic: 'Inventory', payload: 'Test from Inventory' };
```

Comment:
Suggestion: Repetition here can be reduced by having a common message object and differentiating by payload.

Reply:
Will refactor when functionality to consume messages from aws sqs is implemented, just pushed up code to show functionality to publish messages to aws sqs